src/rules/require-baseline.js

```diff
@@ -357,7 +405,17 @@ export default {
 				type: "object",
 				properties: {
 					available: {
-						enum: ["widely", "newly"],
+						anyOf: [
+							{
+								enum: ["widely", "newly"],
+							},
+							{
+								// baseline year
+								type: "integer",
+								minimum: 2000,
+								maximum: new Date().getFullYear() + 1,
```

Comment:
Double-checking: is this intentional? The max now would be 2026?

Or did you mean this?
```suggestion
								maximum: new Date().getFullYear() - 1,
```

Reply:
Ah, yes this was intentional (see https://github.com/eslint/css/pull/81#discussion_r1992486853) but I just realized the maximum is inclusive. My reasoning was that the current year is technically a valid Baseline year, ie https://web.dev/baseline/2025.

Fixed this to use the current year, but let me know if you'd still prefer `YYYY - 1`.

Reply:
Sounds good to me. :+1: